pkg/reconciler/apis/apibinding/apibinding_reconcile.go

```diff
@@ -380,7 +380,7 @@ func (c *controller) reconcileBinding(ctx context.Context, apiBinding *apisv1alp
 		apiBinding.Status.Phase = apisv1alpha1.APIBindingPhaseBound
 	}
 
-	return nil
```

Comment:
this is odd to have it in binding. Would call it from a higher level where the reconcilers are invoked depending on phase.

Reply:
how do you handle the errors then?